src/GeoStatsDevTools.jl

```diff
@@ -60,6 +62,11 @@ include("plotrecipes/solutions/simulation_point_set.jl")
 include("plotrecipes/solutions/estimation_regular_grid.jl")
 include("plotrecipes/solutions/simulation_regular_grid.jl")
 
+
+#post processing
```

Comment:
Try to follow the code conventions, I might be picky, but there is a reason for that :)

In this case, please add a whitespace after the `#` and erase extra blank lines, only one blank line dividing concepts.